Example/SlideController/MainTitleScrollView.swift

```diff
@@ -102,15 +99,18 @@ private extension PrivateMainTitleScrollView {
             constraints.append(view.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -itemOffsetX()))
         }
         NSLayoutConstraint.activate(constraints)
-        internalConstraints.append(contentsOf: constraints)
+    }
+    
+    func removeConstraints(view: UIView) {
+        let viewConstraints = self.constraints.filter({ $0.firstItem === view })
+        self.removeConstraints(viewConstraints)
```

Comment:
per https://developer.apple.com/documentation/uikit/uiview/1622593-removeconstraints?language=objc

> When developing for iOS 8.0 or later, use the NSLayoutConstraint class’s deactivateConstraints: method instead of calling the removeConstraints: method directly. The deactivateConstraints: method automatically removes the constraints from the correct views.